extensions/wikia/UserLogin/js/UserLoginAjaxForm.js

```diff
@@ -93,10 +93,14 @@
 		} else {
 			form = this.form;
 
-			form.slideUp(duration, function () {
+			form.slideUp(duration, (function () {
+				var newForm = content.find('form');
 				form.replaceWith(content);
 				content.slideDown(duration);
-			});
+				newForm.on('submit', this.submitLoginAfterResetPass.bind(this));
+				this.wikiaForm = new WikiaForm(newForm);
```

Comment:
While we are overriding initial form I wanted update all attributes to match current state.